### DISCUSSION TITLE
Multisite (multidomain) and SSG

### DISCUSSION THREAD

**Discussion by thomaskenne** [statamic/cms#4315] [Q&A]
Hi.

I'm running a multi-site Statamic with two domains. I would like to build both sites with SSG and deploy them to different S3 buckets.

It looks like the files for both sites are generated into `storage/app/static`. Is there a way for me to configure where each of the sites are generated to, so I can deploy them two different buckets?

**Comment by duncanmcclean** [comment]
I'm not too familiar with the SSG myself but I believe that if you have the URLs of the sites setup as if they're subdirectories, then they should generate into different folders in the output directory.

Something like `storage/app/static/site1` and `storage/app/static/site2`? You'd then just have to somehow copy those files to the different S3 buckets.

Again, I've not used the SSG much but I presume that's how it works. Other people may be able to share a solid answer.

**Comment by thomaskenne** [comment]
@duncanmcclean It generates the sites in two folders if I change the `url` for the sites in `sites.php`. Unfortunately it prepends the folder name to all of the urls in the generated html.

**Reply by duncanmcclean** [reply]
Oh that's a pain! Hopefully someone else has an another solution, sorry!

**Comment by thomaskenne** [comment]
I have just whipped this together. Seems to be working fine. At least for my use case. 

https://github.com/thomaskenne/ssg/tree/support-multidomain-multisites-in-seperate-output-folders
